recipes/ffmpeg/bld.bat

```diff
@@ -0,0 +1,3 @@
+:: http://ffmpeg.zeranoe.com/blog/?p=300
+
+bash "%RECIPE_DIR%\mingw-w64-build-3.6.0.sh"
```

Comment:
Yeah, I think I am going to do the usual thing here and strip out Windows for now. We can try re-adding when it is a feedstock. It also sounds like MSYS2 environment will be packaged by conda in some usable way. Waiting for that might be the smart strategy to get this working correctly.